docker/ubuntu/Dockerfile

```
@@ -92,7 +94,10 @@ RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        -o cpack_name=dist \
        -o cpack_generator=TGZ \
        -pr "$CONAN_PROFILE" \
-       ${WITH_TESTS_ARG}
+       ${WITH_TESTS_ARG} \
+  && cd .. && tar xf dist.tar.gz \
+  && rm dist.tar.gz
```

No need to change directory.

```suggestion
  && tar xf /root/agent/dist.tar.gz -C /root/agent \
  && rm /root/agent/dist.tar.gz
```